src/os/mixin/renderloop.js

```diff
@@ -9,6 +9,12 @@ goog.require('os.olcs.RenderLoop');
  */
 (function() {
   var origEnable = olcs.AutoRenderLoop.prototype.enable;
+
+  /**
+   * Overridden to listen to <code>os.olcs.RenderLoop.REPAINT</code> events in addition
```

Comment:
I've been using backticks instead of `<code>` in docs for brevity. Either works though, just a side note.

Reply:
Oh, does `js-dossier` support markdown in comments or just the backticks?

Reply:
Yes, it supports markdown. Some details here:
https://github.com/jleyba/js-dossier#formatting